code/go/0chain.net/smartcontract/storagesc/block_reward.go

```diff
@@ -82,17 +85,18 @@ func (ssc *StorageSmartContract) blobberBlockRewards(t *transaction.Transaction,
 			"cannot get all blobbers list: "+err.Error())
 	}
 
-	defer cstate.WithActivation(balances, "hard_fork_1", func() {
-	}, func() {
-		logging.Logger.Info("blobber_block_rewards : cleaning older partition",
-			zap.Any("round", BlobberRewardKey(GetPreviousRewardRound(balances.GetBlock().Round, conf.BlockReward.TriggerPeriod))))
+	defer func() {
+		ferr = cstate.WithActivation(balances, "hard_fork_1", func() {}, func() {
```

Comment:
Same as above, the error in the after function will be shadowed. The `ferr` will be nil even we have an error in the after function.